backend/@types/superagent/index.d.ts

```diff
@@ -0,0 +1,8 @@
+export default {}
+
+declare module 'superagent' {
+  interface Response {
+    req: superagent.Request
+    request: superagent.Request
```

Comment:
Superagent requests return a Response object that includes a reference to the original request. It can be referred to as `req` or `request` and while that may be objectionably inconsistent the codebase does both for purposes of logging and testing.
We import @types/superagent as a dev dependency but someone, somewhere has decided the official library shouldn't include the references to `req` and `request` - search me why. This code tacks them on.

Reply:
ok, presumably you need it then